Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoMeraki.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: Network Session ASIM filtering parser for Cisco Meraki
-  Version: '1.1.1'
-  LastUpdated: Mar 28 2024
+  Version: '1.2.1'
+  LastUpdated: June 5 2024
```

Comment:
"LastUpdated" property exists but is not correct format. The expected format is, for example, "Jun 29, 2024"

Reply:
Done